src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -1029,6 +1034,14 @@ pipelines:
           pattern: previewUrl
           replacement: authoringUrl
         - type: versionFileUpgrader
+    - currentVersion: 3.1.7.3
+      nextVersion: 3.1.9.0
```

Comment:
3.1.9.0 vs 3.1.9.1 and 3.1.9.2 on lines 835 and 836. Two different pipelines yes, but shouldn't they at least be in sync for next version value when it changes. If I remember well it can be only 3.1.9 and not 3.1.9.0 by naming convention.

Reply:
Let's continue with our convention of 3.1.9 and then 3.1.9.1 for the following. This means the last .0 is assumed since we don't always have it. Perhaps this can be revisited in the future to always have .0 at the beginning. I don't have strong feelings about this.

Regarding keeping the pipelines in sync, I don't see that as required nor necessary, it's busy work with no real value.